os/drivers/lcd/Kconfig

```diff
@@ -145,6 +145,48 @@ config LCD_RLANDSCAPE
 
 endchoice
 
+config LCD_ST7701
```

Comment:
Is there any chance to be enabled two lcd drivers? If not, how about using choice to prevent enabling two more LCDs?